provisioning/prelude-manager-oss/post-install.sh

```diff
@@ -448,7 +461,18 @@ EOF
 
 systemctl enable prelude-registrator
 
+systemctl mask firewalld.service
+systemctl enable iptables.service
+systemctl enable ip6tables.service
+
 iptables -I INPUT -p tcp -s 0.0.0.0/0 --dport 4690 -j ACCEPT
 iptables -I INPUT -p tcp -s 0.0.0.0/0 --dport 5553 -j ACCEPT
 
-mkdir -p /var/run/prelude-manager
+#saving iptables rules
+iptables-save > /etc/sysconfig/iptables
+
+systemctl stop firewalld.service
+systemctl start iptables.service
+systemctl start ip6tables.service
```

Comment:
move this to another pull request

Reply:
I don't understand do you want me to close this pull request and create another one? This is the fix, without it the other code is useless. The other code is simply specifying that a service should be started before another service. If you want @ferrarimarco i can create another issue to put the rest of the code but without the lines you requested to move it will not work.